openstack/cronus/ci/test-values.yaml

```diff
@@ -10,7 +10,10 @@ rhea:
     tag: latest
 
 nebula:
+  sentryDsn: 
```

Comment:
This whole file is still failing :) so sentryDsn is still nil, and I have a feeling there are many other fields that are nil.

I having a feeling this CI has always been failing and we just haven't paid attention. But you can also try locally via `helm template --values ci/test-values.yaml` and see where it gives you nil errors